_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,339 @@
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">Nested Lists</h3>
+<div class="example-set">
+    <ul>
+        <li class="item">
+            Apples
+
+            <ul>
+                <li>Red Delicious</li>
+                <li>Macintosh</li>
+                <li>Granny Smith</li>
+                <li>Honey Crisp</li>
+            </ul>
+        </li>
+        <li class="item">Apricots</li>
+        <li class="item">Bananas</li>
+        <li class="item">Blackberries</li>
+    </ul>
+
+    <ol>
+        <li class="item">Blueberries</li>
+        <li class="item">
+            Cherries
+
+            <ol>
+                <li>Bing</li>
+                <li>Balaton</li>
+                <li>Ulster</li>
+                <li>Maraschino</li>
+            </ol>
+        </li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+    </ol>
+</div>
+
+<h3 class="hd-6 example-set-hd">List with alignments</h3>
+<div class="example-set">
+    <ul class="list-unstyled align-left">
+        <li class="item">Apples (left aligned)</li>
+        <li class="item">Apricots (left aligned)</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+            (left aligned)
+        </li>
+        <li class="item">Blackberries (left aligned)</li>
+    </ul>
+
+    <ul class="list-unstyled align-center">
+        <li class="item">Apples (center aligned)</li>
+        <li class="item">Apricots (center aligned)</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+            (center aligned)
+        </li>
+        <li class="item">Blackberries (center aligned)</li>
+    </ul>
+
+    <ul class="list-unstyled align-right">
+        <li class="item">Apples  (right aligned)</li>
+        <li class="item">Apricots  (right aligned)</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+            (right aligned)
+        </li>
+        <li class="item">Blackberries (right aligned)</li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">Bulleted List</h3>
+<div class="example-set">
+    <ul class="list-bulleted">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">Bananas</li>
+        <li class="item">Blackberries</li>
+        <li class="item">Blueberries</li>
+        <li class="item">Cherries</li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+        <li class="item">Figs</li>
+        <li class="item">Oranges</li>
+        <li class="item">Plums</li>
+    </ul>
+
+    <ul class="list-bulleted list-circle">
+        <li class="item">Blueberries</li>
+        <li class="item">Cherries</li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+    </ul>
+
+    <div class="list-bulleted">
```

Comment:
This one should definitely include the [ARIA roles for list](https://github.com/edx/ux-pattern-library/pull/168/files#r48980901) and [ARIA roles for listitem](https://github.com/edx/ux-pattern-library/pull/168/files#r48981088) suggested in previous comments.